.eslintrc.json

```diff
@@ -1,82 +1,109 @@
-        "_browser": "writable",
-        "browser": "readonly"
-    },
-    "ignorePatterns": [
-        "/etc",
-        "build",
-        "dist",
-        "node_modules"
-    ],
-    "overrides": [
-        {
-            "files": ["*.svelte"],
-            "processor": "svelte3/svelte3"
-        }
-    ],
-    "parserOptions": {
-        "ecmaVersion": "latest",
-        "sourceType": "module"
-    },
-    "plugins": ["svelte3"],
-    "rules": {
-        "array-callback-return": "off",
-        "arrow-body-style": "off",
-        "arrow-parens": ["error", "as-needed"],
-        "comma-dangle": ["error", "never"],
-        "consistent-return": "off",
-        "default-case": "off",
-        "eqeqeq": ["error", "smart"],
-        "global-require": "off",
-        "guard-for-in": "warn",
-        "import/extensions": "off",
-        "import/first": "off",
-        "import/no-mutable-exports": "off",
-        "import/no-extraneous-dependencies": ["error", {
-            "devDependencies": true
-        }],
-        "import/prefer-default-export": "off",
-        "import/no-unresolved": "off",
-        "indent": ["error", 4, {"SwitchCase": 1}],
-        "max-len": "off",
-        "no-alert":"off",
-        "no-await-in-loop":"warn",
-        "no-confusing-arrow": ["warn", {
-            "allowParens": true,
-            "onlyOneSimpleParam": true
-        }],
-        "no-console": ["warn", {"allow": ["info", "warn", "error"]}],
-        "no-continue": "off",
-        "no-else-return": ["error", {"allowElseIf": true}],
-        "no-nested-ternary": "off",
-        "no-param-reassign": ["warn", { "props": false }],
-        "no-plusplus": "off",
-        "no-restricted-syntax": "off",
-        "no-return-assign": "off",
-        "no-underscore-dangle":"off",
-        "no-unused-expressions": "off",
-        "no-use-before-define": ["error", {
-            "functions": false,
-            "classes": true,
-            "variables": true,
-            "allowNamedExports": false
-        }],
-        "object-curly-newline": ["error", {"consistent": true}],
-        "object-curly-spacing": ["error", "never"],
-        "one-var": "off",
-        "one-var-declaration-per-line": "off",
-        "prefer-destructuring": "off",
-        "prefer-object-spread": "warn",
-        "quotes": ["error", "double"],
-        "import/no-namespace": "warn"
-    }
+	"env": {
+		"browser": true,
+		"es2021": true,
+		"node": true
+	},
+	"extends": ["prettier"],
+	"globals": {
+		"browser": "readonly",
+		"webkit": "readonly"
+	},
+	"ignorePatterns": ["node_modules", "xcode"],
+	"parserOptions": {
+		"ecmaVersion": 13,
+		"sourceType": "module"
+	},
+	"overrides": [
+		{
+			"files": ["*.svelte"],
+			// why use this instead of the svelte3 plugin?
+			// eslint prettier errors are ignored for stylelint, and...
+			// https://github.com/prettier/eslint-plugin-prettier#svelte-support
+			"parser": "svelte-eslint-parser"
+		}
+	],
+	"plugins": ["prettier"],
+	"rules": {
```

Comment:
There are some rules that valuable to keep with `eslint`. I tried not to duplicate the rules from [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier/blob/main/index.js) that would cause issues.